source/aks/patching-flux-example.html.md.erb

```diff
@@ -24,6 +24,8 @@ You can find info about these components [here](https://fluxcd.io/flux/component
 
 These can be upgraded separately so it's best to use the process detailed below to keep everything in sync.
 
+**NOTE:** If there are API changes listed on the Flux release page, it may be a good idea to leave these until all environments are running the new Flux version to avoid conflicts. Once all environments are upgraded you can test the api changes on plum/toffee before applying to the rest of the apps.
+
```

Comment:
Might be worth adding a note to say that people may see errors in the CI checks but these are usually safe to ignore as it's down the API version the script is looking for

Reply:
Good shout

Reply:
Just thinking, I don't think you should get any CI errors if you do it the way mentioned above. I got them because I was trying to upgrade only in one environment

Reply:
Ah yeah, you're right. As long as the script and apiVersions are all updated at once, it should be fine. Might be worth then just adding a note about the script just to say it might need updated too when updating the versions

Reply:
Couple of links to your PRs might be helpful as well

Reply:
Included Daniel's PRs from previous upgrade as mine were not all following correct method